dependencies.sbt

```diff
@@ -5,3 +5,7 @@ libraryDependencies := Seq(
   "com.typesafe.play" %% "play-slick" % versions.playSlickVersion,
   "com.typesafe.play" %% "play-slick-evolutions" % versions.playSlickVersion
 )
+
+dependencyOverrides ++= Seq(
+  "com.typesafe.play" %% "play" % versions.playVersion
```

Comment:
This fixes the bug where the Docker image contains multiple versions of the framework instead of the one selected by the user.